Additional Istio resources into the reconciliation (#346)
kyma-project/istio: docs/contributor/04-40-istio-resources.md

# Kyma Istio Additional Resources

## Overview

> The Istio additional resources includes Kyma configuration of Istio and consists of:

Reviewer: ```suggestion
The additional Istio resources include the Kyma configuration of Istio. They consist of:
```

> - Istio monitoring configuration details providing Grafana dashboards specification

Reviewer: ```suggestion

- Configuration details for Istio monitoring containing specifications for Grafana dashboards
```

- Istio Ingress Gateway configuring incoming traffic to Kyma
- Mutual TLS (mTLS) configuration enabling mTLS cluster-wide in the STRICT mode

> - Istio [VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/) informing whether Istio is up and running

Reviewer: ```suggestion
- Information about Istio [VirtualService](https://istio.io/docs/reference/config/networking/virtual-service/), which indicates whether Istio is operational.
```
